tests/models.py

```diff
@@ -22,7 +22,7 @@ class Answer(OrderedModel):
         Question, on_delete=models.CASCADE, related_name="answers"
     )
     user = models.ForeignKey(TestUser, on_delete=models.CASCADE, related_name="answers")
-    order_with_respect_to = ("question", "user")
```

Comment:
I think it would better to provide separate test for this

Reply:
Ideally, we should create two identical sets of test cases for the field, one set with the "_id" suffix and the other without it. The current modification is the simplest way to achieve the biggest coverage